README.md

```diff
@@ -37,16 +37,16 @@ with Index 0. That means, that even without a subject interacting in any way wit
 account controlled by their DID. This is a useful transitive property, meaning, everyone with a valid Solana wallet
 `XYZ...` also has a generative DID `did:sol:XYZ...` and a generative Cryptid account `ABC...` (as PDA of DID and Index).
 
-Cryptid Accounts with higher indexes (> 0) are created explicitly and can (optionally) be configured to also adhere to
-Middleware constraints. Middleware Accounts can chain other Middleware accounts allowing for composable solutions 
-without rewriting any (often security-critical) code.
+Cryptid accounts with higher indexes (> 0) are created explicitly and can (optionally) be configured to also adhere to
+Middleware constraints. Middleware accounts can chain other Middleware accounts allowing for composable solutions 
+without rewriting any code. This code is often critical to security and so should be not be changed as a best practice. 
 
 ## Features
 
 * Create a Cryptid account from your existing Solana wallet
 * Access your funds across multiple devices without sharing key material or relying on off-chain infrastructure
-* Cryptid allows you to rotate your key if it becomes compromised, ensuring you can keep control of your identity even
-  if your wallet is breached. Even your initial wallet key can be rotated!
+* Cryptid allows you to remove a key from the DID if it becomes compromised, ensuring you can keep control of your identity even
+  if your wallet is breached. Even your initial wallet key can be removed and replaced with a secure key.
```

Comment:
@rado0x54 just a question on language here... do you prefer to stick to using the term `rotate` or the proposed more descriptive in terms of removing/adding ?

Reply:
My thought with "rotate" is that it is a little vague as to what is required of the user. It also brings to mind the idea of a user being able to put the key back safely at some point (i.e. a baseball pitching rotation, crop rotation, or even rotation of a planet where the planet returns to its original position), which I believe is not the case.